src/python-bindings/pastaq/__init__.py

```diff
@@ -595,7 +597,7 @@ def parse_raw_files(params, output_dir, logger=None, force_override=False):
 
         # Read raw files (MS1).
         _custom_log('Reading MS1: {}'.format(raw_path), logger)
-        if file_extension.lower() == '.mzxml':
+        if raw_path.endswith(".mzXML"):
```

Comment:
I want to make sure that the capitalization of the file extension isn't an issue.

Please revert all changes to the __init__.py file.

Reply:
The changes have been reverted